src/ScottPlot/Diagnostic/CheckFiledsOrderedAccendingDecorator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using ScottPlot.Diagnostic.Attributes;
+using System;
+using System.Linq;
+using System.Reflection;
+
+namespace ScottPlot.Diagnostic
+{
+    public class CheckFiledsOrderedAccendingDecorator : PlottableDecoratorBase
```

Comment:
This is a typo, it should be `CheckFieldsOrderedAscendingDecorator`, although the "Ordered" is probably redundant, I would suggest `CheckFieldsAscendingDecorator`.

Reply:
Your absolutely right. At first i plan to make universal `CheckFieldsOrdered`, but then I realized that in fact only Accending needed, and decided not to complicate things, somehow it happened.